pom.xml

```diff
@@ -49,6 +51,43 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+
+			<dependency>
+				<groupId>commons-logging</groupId>
+				<artifactId>commons-logging</artifactId>
+				<version>1.2</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-sleuth-core</artifactId>
+				<version>${spring-cloud-sleuth.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-starter-sleuth</artifactId>
+				<version>${spring-cloud-sleuth.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>mysql</groupId>
+				<artifactId>mysql-connector-java</artifactId>
+				<version>5.1.42</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.postgresql</groupId>
+				<artifactId>postgresql</artifactId>
+				<version>42.1.1</version>
```

Comment:
In 1.5.x, [we couldn't upgrade](https://github.com/spring-projects/spring-boot/blob/98f577e62c7bd9d95663b0c9c69cc0a4c2d8e788/spring-boot-dependencies/pom.xml#L141) to the next major version of the driver. Is there a reason why you need to stick with one particular version of the driver?

Reply:
No special reason. It was the latest version at the time, but if you're already tracking it, no need for us to.